transformations/gender_randomizer/README.md

```diff
@@ -0,0 +1,45 @@
+# Gender Randomizer 🦎  + ⌨️ → 🐍
+
+Author name: Tabitha Sugumar
+Author email: __
+Author Affiliation: __
+
```

Comment:
Thanks for your changes @tk-sugumar . Please add your email and affiliation.

Reply:
Added!